### PR TITLE
docs: Add tutorial, examples, resources

### PR DESCRIPTION
**Pull request by jnv**

### CODE REVIEW

File: examples/basic-usage/server.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+// Use the Twitter OAuth2 strategy within Passport
+passport.use(
+  new Strategy(
+    {
+      clientID: process.env.TWITTER_CLIENT_ID,
+      clientSecret: process.env.TWITTER_CLIENT_SECRET,
+      clientType: 'confidential',
+      callbackURL: `${process.env.BASE_URL}/auth/twitter/callback`,
+    },
+    // <3> Verify callback
```
Comment (janhalama) [inline examples/basic-usage/server.js:23]: question: I did not find 1, 2, 6 steps. Am I missing something? Also this is not consistent with the second example where you do not have numbered steps.

**Review by janhalama — APPROVED** [review]
Good job! This is major improvement in the documentation 🙏 . I found some minor issues, see my comments.

File: examples/basic-usage/server.js (hunk 1 above)
Comment (jnv) [reply examples/basic-usage/server.js:23]: Sorry, these were leftovers from my tutorial, I will remove the numbers everywhere.

**Comment by jnv** [comment]
Sorry, I saw you already preapproved the PR, I will merge it.

**Pull request merged**